status-go-version.json

```diff
@@ -2,7 +2,7 @@
     "_comment": "DO NOT EDIT THIS FILE BY HAND. USE 'scripts/update-status-go.sh <tag>' instead",
     "owner": "status-im",
     "repo": "status-go",
-    "version": "v0.42.0",
-    "commit-sha1": "9cf640842b4d0266bcc1b30f6d776c250fdb5c04",
-    "src-sha256": "0h8da2w3fbgrv6rmd4pib46r97g5q2wwg4a790f91dadsl67pmcl"
+    "version": "v0.41.1",
```

Comment:
in develop it's 42, should it be 42.1 ?

Reply:
It's a completely different version, not associated to the tag. (PR in status-go is not merged yet) https://github.com/status-im/status-go/pull/1832 , I will update to the correct version once it gets in